### PR TITLE
Add MILP-based evaluation of AEGIS

### LINKED ISSUE #71: Another publication on AEGIS
MILP-based security evaluation for AEGIS/Tiaoxin-346/Rocca: https://ietresearch.onlinelibrary.wiley.com/doi/full/10.1049/ise2.12109

### PR DESCRIPTION
**Pull request by jedisct1**
Fixes #71

### CODE REVIEW

**Comment by jedisct1** [comment]
Martin's scripts to build HTML files seems to be currently broken :/

File: draft-irtf-cfrg-aegis-aead.md [hunk 1]
```diff
@@ -884,7 +909,7 @@ With AEGIS-256, random nonces can be used with no practical limits.
 
 The security of AEGIS against timing and physical attacks is limited by the implementation of the underlying `AESRound()` function. Failure to implement `AESRound()` in a fashion safe against timing and physical attacks, such as differential power analysis, timing analysis or fault injection attacks, may lead to leakage of secret key material or state information. The exact mitigations required for timing and physical attacks also depend on the threat model in question.
 
-Security analyses of AEGIS can be found in Chapter 4 of {{AEGIS}}, in {{Min14}}, in {{ENP19}}, in {{LIMS21}}, and in {{JLD21}}.
+Security analyses of AEGIS can be found in Chapter 4 of {{AEGIS}}, in {{Min14}}, in {{ENP19}}, in {{LIMS21}}, in {{JLD21}}, and in {{SHI23}}.
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md:912]: Think this should be `{{STSI23}}` for Shiraya, Takeuchi, Sakamoto, and Isobe.

File: draft-irtf-cfrg-aegis-aead.md [hunk 2]
```diff
@@ -156,6 +156,31 @@ informative:
         org: Agence nationale de la sécurité des systèmes d'information ANSSI
     date: 2014
 
+  SHI23:
+    title: "MILP-based security evaluation for AEGIS/Tiaoxin-346/Rocca"
+    rc: "Institution of Engineering and Technology"
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md:161]: Should this be 'IET Information Security'?

File: draft-irtf-cfrg-aegis-aead.md [hunk 3]
```diff
@@ -156,6 +156,31 @@ informative:
...
+        name: Nobuyuki Takeuchi
+        org: University of Hyogo, Kobe, Japan.
+      -
+        ins: K. Sakamoto
+        name: Kosei Sakamoto
+        org: University of Hyogo, Kobe, Japan.
+      -
+        ins: T. Isobe
+        name: Takanori Isobe
+        org: University of Hyogo, Kobe, Japan.
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md:181]: Previously also had '; National Institute of Information and Communications Technology' if you look at `LIMS21`.

File: draft-irtf-cfrg-aegis-aead.md [hunk 4]
```diff
@@ -156,6 +156,31 @@ informative:
...
+    title: "MILP-based security evaluation for AEGIS/Tiaoxin-346/Rocca"
+    rc: "Institution of Engineering and Technology"
+    seriesinfo:
+      DOI: 10.1007/978-3-319-93387-0_25
+    target: https://ietresearch.onlinelibrary.wiley.com/doi/10.1049/ise2.12109
+    author:
+      -
+        ins: T. Shiraya
+        name: Takuro Shiraya
+        org: University of Hyogo, Kobe, Japan.
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md:169]: Don't think ', Kobe, Japan.' is needed based on previous references (e.g. `LIMS21`).

**Review by samuel-lucas6 — CHANGES_REQUESTED** [review]

**Comment by jedisct1** [comment]
Thank you Samuel!

These changes have been made.

File: draft-irtf-cfrg-aegis-aead.md (hunk 2 above)
Comment (samuel-lucas6) [reply draft-irtf-cfrg-aegis-aead.md:161]: Sorry to be annoying, but I think this should be "IET Information Security, 2023, pp. 1-10" based on [this](https://ietresearch.onlinelibrary.wiley.com/action/showCitFormats?doi=10.1049%2Fise2.12109) and the other rcs.

File: draft-irtf-cfrg-aegis-aead.md [hunk 5]
```diff
@@ -156,29 +156,29 @@ informative:
...
       -
         ins: K. Sakamoto
         name: Kosei Sakamoto
-        org: University of Hyogo, Kobe, Japan.
+        org: University of Hyogo
       -
         ins: T. Isobe
         name: Takanori Isobe
-        org: University of Hyogo, Kobe, Japan.
+        org: University of Hyogo; National Institute of Information and Communications Technology; PRESTO, Japan Science and Technology Agency
```
Comment (samuel-lucas6) [inline draft-irtf-cfrg-aegis-aead.md:181]: '; PRESTO, Japan Science and Technology Agency' can be removed as it isn't mentioned in this paper.

**Review by samuel-lucas6 — CHANGES_REQUESTED** [review]
Thanks for the changes. Two little things.

There are a few inconsistencies for the other references (e.g. no target, no page numbers, specific vs year for dates) that can probably be tidied up in a separate PR.

File: draft-irtf-cfrg-aegis-aead.md (hunk 5 above)
Comment (jedisct1) [reply draft-irtf-cfrg-aegis-aead.md:181]: Thanks Samuel!

Chronologically ordering the references may also be a good thing to do, but that can be done in another PR.

**Pull request merged**